src/settings/utils/filtering.js

```diff
@@ -0,0 +1,3 @@
+export const extractSelectedIdsFromObject = (object) => Object.entries(object)
+  .filter(([, isSelected]) => isSelected)
+  .map(([id]) => id);
```

Comment:
Please include a header comment, maybe sth like "This function takes an object and returns a list of the keys with truthy values." Based on its name, I expected it to do ... I don't know what exactly, maybe take some kind of `isSelected` predicate, or for there to be an array somewhere with an `isSelected` and `id` properties and to filter based on those. Anyway. What it does is totally fine, but please include an explanation of what it does.